From: Gavin Lambert <gavin.lambert@tomra.com>
Date: Wed, 26 Jul 2017 12:28:21 +1200

Increment interface versions.

This is a placeholder assuming that you will be applying patches that change
the ABI for both ecrt and ioctl interfaces.  It should not be applied upstream
as-is, and if you are only applying subsets of patches at a time (and you want
to know when you have mismatched library and app code and need to recompile)
then you may want to increment the versions as you apply different sets.

diff --git a/include/ecrt.h b/include/ecrt.h
--- a/include/ecrt.h
+++ b/include/ecrt.h
@@ -142,13 +142,17 @@
  */
 #define ECRT_VER_MINOR 5
 
+/** EtherCAT realtime interface patchlevel number.
+ */
+#define ECRT_VER_PATCH 10
+
 /** EtherCAT realtime interface version word generator.
  */
-#define ECRT_VERSION(a, b) (((a) << 8) + (b))
+#define ECRT_VERSION(a, b, c) (((a) << 16) + ((b) << 8) + (c))
 
 /** EtherCAT realtime interface version word.
  */
-#define ECRT_VERSION_MAGIC ECRT_VERSION(ECRT_VER_MAJOR, ECRT_VER_MINOR)
+#define ECRT_VERSION_MAGIC ECRT_VERSION(ECRT_VER_MAJOR, ECRT_VER_MINOR, ECRT_VER_PATCH)
 
 /******************************************************************************
  * Feature flags
diff --git a/master/ioctl.h b/master/ioctl.h
--- a/master/ioctl.h
+++ b/master/ioctl.h
@@ -56,7 +56,7 @@
  *
  * Increment this when changing the ioctl interface!
  */
-#define EC_IOCTL_VERSION_MAGIC 30
+#define EC_IOCTL_VERSION_MAGIC 36
 
 // Command-line tool
 #define EC_IOCTL_MODULE                EC_IOR(0x00, ec_ioctl_module_t)
